poe-api/src/poe.py

```diff
@@ -518,6 +550,27 @@ def send_message(self, chatbot, message, with_chat_break=False, timeout=20):
       message_id = message["messageId"]
 
       yield message
+    
+    # wait 2 senconds after sending the request
+    time.sleep(2)
```

Comment:
Seems kind of pointless to make all the waiting in this function be blocking. This could easily be moved to a new thread. I'll probably implement that on my own though.